code/drasil-printers/lib/Drasil/Database/SearchTools.hs

```diff
@@ -0,0 +1,77 @@
+module Drasil.Database.SearchTools where
+
+import Control.Lens ((^.))
+
+import Database.Drasil
+import Language.Drasil
+import Theory.Drasil
+
+-- This is only neede for as long as `TypedUIDRef` is underused.
+data TermAbbr = TermAbbr { longForm :: NP, shortForm :: Maybe String }
```

Comment:
TODO: @balacij Typo.

Also, the comment explains itself. This is only necessary for as long as we don't use `TypedUIDRef`.

Reply:
Issue is fixed in bc94e39b7353187d03fad3f976bbbbe56f4c6765. Leaving this comment unresolved as it also contains commentary.